vnext/local-cli/runWindows/utils/deploy.js

```diff
@@ -214,21 +209,21 @@ async function deployToDesktop(options, verbose, slnFile) {
     verbose,
   );
 
-  // #4749 - need to deploy from appx for .net projects.
-  if (options.release || (await hasDotNetProjects(slnFile))) {
+  if (options.release) {
     await runPowerShellScriptFunction(
       'Installing new version of the app',
       windowsStoreAppUtils,
       `Install-App "${script}" -Force`,
       verbose,
     );
   } else {
-    const realAppxManifestPath = fs.realpathSync(appxManifestPath);
-    await runPowerShellScriptFunction(
-      'Installing new version of the app from layout',
-      windowsStoreAppUtils,
-      `Install-AppFromDirectory "${realAppxManifestPath}"`,
-      verbose,
+    await build.buildSolution(
+      slnFile,
+      options.release ? 'Release' : 'Debug',
```

Comment:
>release ? 'Relea [](start = 14, length = 16)

nitpick: This is fine and defense for future, but technically it can never be release because it is in the else block :)

Reply:
yeah I figured better safe than sorry to not try to outsmart the language by pre-optimizing :)